core-bundle/src/DependencyInjection/Security/ContaoLoginFactory.php

```diff
@@ -105,4 +110,33 @@ private function createTwoFactorPreparationListener(ContainerBuilder $container,
             ->addTag('kernel.event_listener', ['event' => 'kernel.finish_request', 'method' => 'onKernelFinishRequest'])
         ;
     }
+
+    private function createTwoFactorTrustedDevicesTokenStorage(ContainerBuilder $container, string $firewallName): void
+    {
+        $container
+            ->setDefinition('contao.security.two_factor.trusted_token_storage', new ChildDefinition('scheb_two_factor.trusted_token_storage'))
+            ->replaceArgument(2, self::TRUSTED_DEVICES_TOKEN_ID_PREFIX)
+            ->replaceArgument(3, self::TRUSTED_DEVICES_TOKEN_LIFETIME)
+        ;
+    }
+
+    private function createTwoFactorTrustedCookieResponseListener(ContainerBuilder $container, string $firewallName): void
+    {
+        $trustedCookieResponseListenerId = 'contao.listener.two_factor.trusted_cookie_response_listener.'.$firewallName;
+        $domain = '/';
+
+        if ('contao_backend' === $firewallName) {
+            $domain = '/contao';
+        }
+
+        $container
+            ->setDefinition($trustedCookieResponseListenerId, new ChildDefinition('scheb_two_factor.trusted_cookie_response_listener'))
+            ->replaceArgument(0, new Reference('contao.security.two_factor.trusted_token_storage'))
+            ->replaceArgument(1, self::TRUSTED_DEVICES_TOKEN_LIFETIME)
+            ->replaceArgument(2, self::TRUSTED_DEVICES_TOKEN_ID_PREFIX)
+            ->replaceArgument(5, $domain)
+            ->addTag('kernel.event_listener', ['event' => 'kernel.response', 'method' => 'onKernelResponse'])
```

Comment:
The service definition already has the event listener by default, no need to add it here.

Reply:
Fixed in c25176a8f.